deps/wazuh_testing/wazuh_testing/tools/remoted_sim.py

```diff
@@ -107,38 +112,47 @@ def _start_socket(self):
             self.sock.bind((self.server_address, self.remoted_port))
 
     def set_wcom_message_version(self, version):
-        """
-        Set version for WPK tests
+        """Set version for WPK tests.
+
+        Args:
+            version (str): WPK version.
         """
         self.wcom_message_version = version
 
     def set_active_response_message(self, ar_message):
-        """
-        Set message for AR tests
+        """Set message for AR tests.
+
+        Args:
+            ar_message (str): Active response message.
         """
         self.active_response_message = ar_message
 
     def stop(self):
-        """
-        Stop socket and listener thread
-        """
+        """Stop socket and listener thread"""
         if self.running:
             self.running = False
             self.listener_thread.join()
             self.sock.close()
 
     def create_encryption_key(self, agent_id, name, key):
-        """
-        Generate encryption key (using agent metadata and key)
+        """Generate encryption key (using agent metadata and key).
+
+        Args:
+            agent_id (str): Agent id.
+            name (str): Agent name.
+            key (str): Encryption key.
         """
         sum1 = (hashlib.md5((hashlib.md5(name.encode()).hexdigest().encode() + hashlib.md5(
             agent_id.encode()).hexdigest().encode())).hexdigest().encode())[:15]
         sum2 = hashlib.md5(key.encode()).hexdigest().encode()
         self.encryption_key = sum2 + sum1
 
     def compose_sec_message(self, message, binary_data=None):
-        """
-        Compose event from raw message
+        """Compose event from raw message.
+
+        Args:
+            message (str): Raw message.
+            binary_data(str): Binary data.
```

Comment:
```suggestion
            binary_data (str): Binary data.
```
Also, this doc string... Is a bit obvious. Can you give more details about this variable?